testing/test_cluster.py

```diff
@@ -38,6 +39,29 @@ def make_cluster(self, num_nodes):
                                                             self.ramcloud_network)
         self.rc_client.connect(external_storage, 'main')
 
+    def simple_recovery(self, use_force):
+        self.make_cluster(num_nodes=7)
+        self.createTestValue()
+        value = self.rc_client.read(self.table, 'testKey')
+        expect(value).equals(('testValue', 1))
+
+        # find the host corresponding to the server with our table and 'testKey',
+        # then find the server's pid, and kill it!
+        locator =  self.rc_client.testing_get_service_locator(self.table, 'testKey')
+        host = ctu.get_host(locator)
+        processes = self.node_containers[host].top()['Processes']
```

Comment:
As an alternative to using `container.top()` to get a list of processes and then filtering them, we could install the [psmisc](https://packages.debian.org/buster/psmisc) Debian package by adding it to the `apt-get install` list in `./config/Dockerfile.node`. Then we can simply issue a `killall -TERM rc_server` for a graceful down and `killall -KILL rc_server` for a forced down. The command can probably be issued like `self.node_containers[host].exec_run(cmd='killall -TERM rc_server')`. At any rate, please use `-TERM` and `-KILL` rather than nothing (for `SIGTERM`) and `-9` for `SIGKILL` because it is far more readable.

Reply:
Using -SIGTERM and -SIGKILL. These worked for me at command line